src/component.js

```diff
@@ -176,10 +175,7 @@ let rerenderQueue = [];
  */
 /* istanbul ignore next */
 // Note the following line isn't tree-shaken by rollup cuz of rollup/rollup#2566
-const defer =
-	typeof Promise == 'function'
-		? Promise.prototype.then.bind(Promise.resolve())
-		: setTimeout;
+const defer = Promise.prototype.then.bind(Promise.resolve());
```

Comment:
Wouldn't `queueMicrotask` also be a possibility here?

Reply:
Good catch. Browser support is looking really good for that too. What do you think @developit ?

Reply:
I had thought it was worse, but yeah you're right, it's decent. Main concern is Edge 16-18 and Safari 12, but those are looking pretty old these days...